Automattic/newspack-theme
# fix: update theme SCSS file formatting

### All Submissions:


### Changes proposed in this Pull Request:

This PR restores the new `@use` approach in the themes' SCSS introduced in #1845.

These changes were rolled back when the [latest release was merged in](https://github.com/Automattic/newspack-theme/commit/ff56ad9553683f3a88116b0a29460dbd006fb986#diff-853fc951e3761daa8c9be32a503af30fa17b0f4b5ea66aaef0688b1baba2ce9c).

Note that each theme now includes a separate theme-description.scss file ([example in parent](https://github.com/Automattic/newspack-theme/blob/master/newspack-theme/sass/theme-description.scss) and [example in a child theme](https://github.com/Automattic/newspack-theme/blob/master/newspack-joseph/sass/theme-description.scss)). I've manually bumped these to match the latest release (1.61.0) but more work may need to be done to get these bumping as part of the release cycle -- if that's the case please let me know! 

### How to test the changes in this Pull Request:

1. Try running `npm run build` on master and note the build errors.
2. Apply the PR and run `npm run build`; confirm that the CSS is generated without error. 
3. Spot check the file changes and confirm they essentially undo non-release changes [in this PR](https://github.com/Automattic/newspack-theme/commit/ff56ad9553683f3a88116b0a29460dbd006fb986): the top of each theme's style.scss should be switched from a description to files loaded with `@import`, to just files loaded with `@use`.

### Other information:
